simple_qa_rnn/README.md

```diff
@@ -1,14 +1,14 @@
-Directions:
-1. Download SimpleQuestions data from this [link](https://www.dropbox.com/s/tohrsllcfy7rch4/SimpleQuestions_v2.tgz) and put it in directory "data/SimpleQuestions_v2/"
-2. Run this script to download GloVe word embeddings and do some preprocessing.
+## Relation Prediction Model
+
+- Download and extract SimpleQuestions dataset by running the script:
 ```
-bash fetch_and_preprocess.sh
+bash fetch_dataset.sh 
 ```
-3. Run this command to train the model. Make sure you have PyTorch and other Python dependencies installed.
+
+- Run the training script with the following commands. Please check out args.py file to see the different commands available:
 ```
-python train_relation_model.py 
+cd relation_prediction
+python train.py
+python train.py --cuda
```

Comment:
Maybe we should use consider using cuda by default - hence `no-cuda` should be the parameter. I didn't do this for sm_model but will change it. See https://github.com/pytorch/examples/blob/53f25e0d0e2710878449900e1e61d31d34b63a9d/mnist/main.py#L29 for example.